proto/proto/store.proto

```diff
@@ -1,20 +1,22 @@
-// Specification of the database RPC.
+// Specification of the store RPC.
 //
 // This provided access to the rollup data to the other nodes.
 syntax = "proto3";
 package store;
 
 import "digest.proto";
+import "tsmt.proto";
 
 message CheckNullifiersRequest {
-    digest.Digest nullifier = 1;
+    repeated digest.Digest nullifiers = 1;
 }
 
 message CheckNullifiersResponse {
-    bool result = 1;
+    // Each requested nullifier has its corresponding nullifier proof at the
+    // same position.
+    repeated tsmt.Proof proofs = 1;
 }
```

Comment:
Should we use a more specialized message here - e.g., something like `NullifierProof`? One reason is that for nullifiers we don't need to user a full word for `value` - just a `u32` should be enough.

Reply:
changed